src/CosmosDB/Microsoft.Azure.Workflows.ServiceProvider.Extensions.CosmosDB.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\sign.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.CosmosDB" Version="3.0.3" />
+    <PackageReference Include="Microsoft.Azure.Workflows.WebJobs.Extension" Version="*-*" />
```

Comment:
can we have the specific (latest) version here  (maybe have a tie it to a partcular minor number which we can increment whenever there is a breaking change)- otherwise any breaking change there would break this repo - we can continue to update whenever we h